helm/prometheus-chart/values.yaml

```diff
@@ -449,7 +449,7 @@ initChownData:
   ## If false, data ownership will not be reset at startup
   ## This allows the prometheus-server to be run with an arbitrary user
   ##
-  enabled: true
+  enabled: false
```

Comment:
Shouldn't all of these be true?

Reply:
Yeah I will like to default everything to enable, you can always customize your values and use them in the installation command. 
```
helm install -f myvalues.yaml
```